include/translatable-object.php

```diff
@@ -485,20 +485,27 @@ public function set_language_in_mass( $ids, $lang ) {
 		if ( empty( $tt_id ) ) {
 			return;
 		}
+		$ids = array_map( 'intval', $ids );
```

Comment:
Maybe the sanitization should be done outside `set_language_in_mass()` according to the documentation.

Reply:
That's how @Chouby reassures himself 😏

Reply:
I see 😅